edgelet/edgelet-kube/src/convert/to_k8s.rs

```diff
@@ -540,8 +550,17 @@ mod tests {
     use edgelet_docker::DockerConfig;
     use edgelet_test_utils::cert::TestCert;
 
-    use crate::constants::env::*;
-    use crate::constants::*;
+    use crate::constants::env::{
```

Comment:
nit: I'd say we use this constants via `env::CONSTANT_NAME` instead of importing all of them

Reply:
I think all these constants are used in this file.  Do you mean that we should state full qualified name like "crate::constants::env::<constant name>" in every usage?

Reply:
it's a nit, so you can leave it here. it's just constants that may have the same name because they live if different modules, but importing it like this can lead to a name collision.
it's not the case here though

```rust
// I meant
use crate::contants::env;

...

// later in the code use it like

foo(env::CONSTANT_NAME);
```